### ISSUE TITLE
L4T 35.4.1 tegraparser_v2 returns error on zero length DATAFILE

### ISSUE THREAD

**Issue by Lexmark-chad** [madisongh/digsigserver#20]
TL;DR;

There appears to be an error, but everything seems to work OK.

See [this](https://forums.developer.nvidia.com/t/tegraparser-v2-behaves-differently-between-l4t-35-4-1-and-l4t-32-7-4/272872) forum post for context.

An error will be shown in `digsigserver`'s logs although the generation of `flash.idx` is the last step in the signing functions within `tegraflash_internal.py`.  No error codes are captured and returned so `digsigserver` is going to return success - `200`.  The resulting packaging will be missing `flash.idx`, but I'm not sure what it is used for.  As I was chasing issues with getting signing completely working with L4T 35.4.1 tooling I thought this was the source of one of my issues so I crafted a 'fix' in [this branch](https://github.com/Lexmark-chad/meta-tegra/tree/fix-datafile-zero-length) to the `tegraXXX-flash-helper` scripts.  Turns out that doesn't really fix anything other than addressing the superficial error in the logs.

Worth fixing?  Ignore?  I think it's worth fixing so that this error doesn't lead to future chasing of red herrings.

**Comment by madisongh** [comment]
What we probably need to do is update the helper scripts to remove those `touch` commands and instead remove the `DATAFILE` and other problematic file references in a temp copy of the layout.  Something along those lines, at least.
